.gitignore

```diff
@@ -9,6 +9,8 @@
 .env*
 .npm
 .gitlab
+.DS_Store
+.idea
```

Comment:
Because of the Webstorm (IDE), that i am using.

Reply:
👍